paprika_recipes/commands/download_recipes.py

```diff
@@ -16,6 +16,7 @@ def get_help(cls) -> str:
     @classmethod
     def add_arguments(cls, parser: argparse.ArgumentParser, config: ConfigDict) -> None:
         parser.add_argument("export_path", type=Path)
+        parser.add_argument("--download_images", default=False, type=bool, help="Allows to download every image attached to the recipes")
```

Comment:
```suggestion
        parser.add_argument("--download-images", default=False, type=bool, help="Downloads images attached to the recipes")
```

Reply:
Good idea! I'll make the change, sadly the inline  suggestion no longer work